entry/render/buildRenderEntry.js

```diff
@@ -0,0 +1,22 @@
+/* eslint-disable import/no-unresolved */
```

Comment:
This is operating on the whole file. Should we scope it to the `'__sku_alias__renderEntry'` import?